### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by SergSkol**
- Add JS module ToDoList;
- Add array of objects for tasks;
- Add methods to add, remove, show;
- Add CSS;

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/* eslint-disable */
```
Comment (christianbiring1) [inline server.js]: - [ ] Kindly do not disable the linters as they help standardize your code base.

File: src/modules/ToDoList.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  }
+
+  addTask() {
+    const toDoAdd = document.querySelector('#add');
+    const newItem = {
+      index: Date.now(),
+      description: toDoAdd.value,
+      completed: false,
+    };
+
```
Comment (christianbiring1) [inline src/modules/ToDoList.js:14]: - [ ] As said in the project requirements, Kindly make sure the todo tasks are retrieved for an array of objects and rendered on the page when it loads. You may consider adding the arrays of tasks.
As you can see your App on it rendering :point_down: 
image

Comment (christianbiring1) [reply src/modules/ToDoList.js:14]: - [ ] On page load, the task array is rendered dynamically to the DOM. The tasks should appear in order of the index values for each task.

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @SergSkol , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose


## Highlights
- [ ] Webpack was set correctly :+1: 
- [ ] Descriptive PR `title` and `summary` :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by prolajumokeoni** [comment]
Hi @SergSkol  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that  has EsLint errors, please fix it 👍🏿 
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by christianbiring1 — APPROVED** [review]
Hi @SergSkol , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
imageswelljob

## Highlights
- [ ] All linters are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] Good commits messages.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**